src/script/standard.cpp

```diff
@@ -256,7 +256,8 @@ class CScriptVisitor : public boost::static_visitor<bool>
 private:
     CScript *script;
 public:
-    explicit CScriptVisitor(CScript *scriptin) { script = scriptin; }
+    explicit CScriptVisitor(CScript *scriptin) : script(scriptin) {
+    }
```

Comment:
nit: whitespace

Reply:
@Empact Where do you want the whitespace added/removed? :-)

Reply:
Probably leave everything in a single line?

Reply:
`clang-tidy` accepts at least:
`explicit CScriptVisitor(CScript* scriptin) : script(scriptin) {}` or:
```
explicit CScriptVisitor(CScript* scriptin) : script(scriptin)
{
}
```

For an empty set, like Marco I like the former.

Reply:
Could also apply to `CompareInvMempoolOrder`, `FreespaceChecker`, etc.